frontend/src/components/VAudioThumbnail/VAudioThumbnail.vue

```diff
@@ -36,12 +36,13 @@
 </template>
 
 <script lang="ts">
+import { useI18n } from "#imports"
```

Comment:
This syntax is new.

Reply:
Do you mean the `#imports`, @dhruvkb ?

Reply:
This is the first time I've seen that, too! 😮